src/Comment.jsx

```diff
@@ -220,8 +213,12 @@ class _Comment extends React.Component {
         </div>
     );
 
+    tt(key) {
+        const {t, locale} = this.props;
+        return t(key, {lng: locale});
```

Comment:
Rely on what is passed to the component and not the language of the i18n instance.

Reply:
Interestingly, when I pass `locale` to the component it sets the language, even without these changes.